adabot_description/urdf/adabot.macros.xacro

```diff
@@ -161,10 +128,28 @@
             total  : the number of wegs (used to calculate angles)
     -->
     <xacro:macro name="weg" params="wheel number parent total">
-
+        
         <!-- The name of the current weg (based on the parent wheel's name) -->
         <xacro:property name="weg_name" value="${wheel}_weg_${number}" />
 
+        <!-- Create the weg geometry (for visuals and collisions) -->
+        <xacro:property name="weg_geometry">
+            <origin xyz="${wh_radius/2} 0 0" rpy="0 0 0" />
+            <!-- <origin xyz="${wh_radius} 0 0" rpy="0 0 0" /> -->
+            <geometry>
+                <box size="${wg_length} ${wg_width} ${wg_height}" />
+            </geometry>
+        </xacro:property>
+        
+        <!-- Create a geometry for the weg tip -->
+        <xacro:property name="weg_tip_geometry">
+            <!-- <origin xyz="${wh_radius} 0 0" rpy="0 0 0" /> -->
```

Comment:
Weg tips are either spheres (`debug_visuals:=true`) or they are not displayed (they are taken care of by the weg meshes).